app/messaging/index.js

```diff
@@ -36,12 +37,24 @@ async function sendRequestSBIMessage (payload, correlationId) {
   console.info('SBI(s) requested')
 }
 
+async function receiveStandardsResponseMessage (messageId) {
+  console.info('Response Standards requested')
```

Comment:
Suggest removal of this log line as they just flood the logs in the cluster making it difficult to pick out meaningful messages

Reply:
Agree